worker/introspection/socket_test.go

```diff
@@ -107,7 +107,7 @@ func (s *introspectionSuite) TestCmdLine(c *gc.C) {
 }
 
 func (s *introspectionSuite) TestGoroutineProfile(c *gc.C) {
-	buf := s.call(c, "/debug/pprof/goroutine")
+	buf := s.call(c, "/debug/pprof/goroutine?debug=1")
```

Comment:
> was this for debugging purposes, what happens on a go 1.6 version if we pass these? Given that the URL is 'debug' it feels weird to have to specify debug=1 again.

There's two levels of debugging here :)
/debug/... is for debugging your application; ?debug is for debugging that output.

This is just to force the textual representation, for backwards compatibility. Go 1.6 does not care about the additional `?debug=1`. When we raise the minimum version of Go for running the tests to Go 1.8, then we can check the new (binary) format, and drop this.